Bottom: 0f2c63215b46f2e7b03a0c519b778a3768ce08e6
Top:    a1fd8aada22180dbaea6ccd8040192f19dfd94a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-23 23:53:21 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
